recipe/meta.yaml

```diff
@@ -22,29 +22,35 @@ requirements:
     - wheel
```

Comment:
Do we need `setuptools_scm >=8`? See https://github.com/scikit-learn-contrib/imbalanced-learn/blob/0.13.0/pyproject.toml#L2C31-L2C54

Reply:
Thanks for catching that!